README.md

```diff
@@ -131,85 +131,83 @@ import { PortableText, PortableTextReactComponents } from "@portabletext/react";
+        return <div>{item?.elements.text_element.value}</div>;
+      },
+      table: ({ value }) => {
+        const table = (
+          <table>
+            {value.rows.map((row) => (
+              <tr>
+                {row.cells.map((cell) => {
+                  return (
+                    <td>
+                      <PortableText
+                        value={cell.content}
+                        components={portableTextComponents}
+                      />
+                    </td>
+                  );
+                })}
+              </tr>
+            ))}
+          </table>
+        );
+        return table;
+      },
+      image: ({ value }) => {
+        // It is possible to use images from the rich text element response same as for linked items
+        // const image = images.find(image => image.image_id === value.asset._ref)
+        return <img src={value.asset.url}></img>;
+      },
     },
-    internalLink: ({ value, children }) => {
-      // It is possible to use links from the rich text element response same as for linked items
-      // const item = links.find(link => link.link_id === value.reference._ref);
-      return (
-        <a href={"https://somerandomwebsite.xyz/" + value.reference._ref}>
-          {children}
-        </a>
-      );
+    marks: {
+      link: ({ value, children }) => {
+        const target = (value?.href || "").startsWith("http")
+          ? "_blank"
+          : undefined;
+        return (
+          <a
+            href={value?.href}
+            target={target}
+            rel={value?.rel}
+            title={value?.title}
+            data-new-window={value["data-new-window"]}
+          >
+            {children}
+          </a>
+        );
+      },
+      internalLink: ({ value, children }) => {
+        // It is possible to use links from the rich text element response same as for linked items
+        // const item = links.find(link => link.link_id === value.reference._ref);
+        return (
+          <a href={"https://somerandomwebsite.xyz/" + value.reference._ref}>
+            {children}
+          </a>
+        );
+      },
     },
-  },
-});
+  };
 
-const MyComponent = ({ props }) => {
-  // https://github.com/portabletext/react-portabletext#customizing-components
-  const portableTextComponents = useMemo(
-    () => createPortableTextComponents(props.element.linkedItems),
-    [props.element.linkedItems]
-  );
+  const MyComponent = ({ props }) => {
+    // https://github.com/portabletext/react-portabletext#customizing-components
+    const parsedTree = browserParse(props.element.value);
+    const portableText = transformToPortableText(parsedTree);
 
-  const parsedTree = browserParse(props.element.value);
-  const portableText = transformToPortableText(parsedTree);
-
-  return (
-    <PortableText value={portableText} components={portableTextComponents} />
-  );
+    return (
+      <PortableText value={portableText} components={portableTextComponents} />
```

Comment:
`portableTextComponents` are note defined - this is the practice by react portable text package..